frontend/src/pages/Register.js

```diff
@@ -0,0 +1,55 @@
+import '../styles/Register.css';
+import React, { useEffect, useState } from "react";
+import { Input } from 'semantic-ui-react'
+import { Button } from 'semantic-ui-react'
+import { register } from '../utils/api.js'
+
+const InputExampleInput = () =>
+<div>  
+    <Input size='large' icon='username' placeholder='Username' />
+    <br />
+    <br />
+    <Input size='large' icon='password' placeholder='Password' /> 
+</div> 
+
+const ButtonExampleShorthand = () => <Button content='Login' />
+
+function LoginPage(props) {
+   const [name, setName] = useState("");
+   const [username, setUsername] = useState("");
+   const [password, setPassword] = useState("");
+
+  return (
+    <div>
+        
+      <h1>Roomble</h1>
+      <div class="rectangle"> 
+      <div class="input">
```

Comment:
change the "input" class to a different className cuz I think this class is used by the <input> tag and that might be why the input cursor is messed up